File: Solution in C++/Assignment_02/Task22.cpp
#include<iostream>

using namespace std;

int main(){
    int a,b,total;
    cin>>a>>b;
    total = a*120 + b*75;
    if(total<0){cout<<"Total can't be negative";}
    else if(total<300){cout<<"Previous total: "<<total<<endl<<
        "No discount"<<endl;
    }
    else if(total<500){cout<<"Previous total: "<<total<<endl<<
        "New total after discount: "<<total-10<<endl;
    }
    else if(total<750){cout<<"Previous total: "<<total<<endl<<
        "New total after discount: "<<total-20<<endl;
    }
    else if(total<1000){cout<<"Previous total: "<<total<<endl<<
        "New total after discount: "<<total-50<<endl;
    }
    else{cout<<"Previous total: "<<total<<endl<<
        "New total after discount: "<<total-150<<endl;}
}

File: Solution by Python/Assignment_02/Task22.py
m=int(input())
n=int(input())
tt=(m*120)+(n*75)
if tt<0:
    print("Invalid")
elif tt<300:
    print(f"Previous total: {tt} \nNew total after discount: {tt}")
elif tt<500:
    print(f"Previous total: {tt} \nNew total after discount: {tt-10}")
elif tt<750:
    print(f"Previous total: {tt} \nNew total after discount: {tt-20}")
elif tt<100:
    print(f"Previous total: {tt} \nNew total after discount: {tt-50}")
else:
    print(f"Previous total: {tt} \nNew total after discount: {tt-150}")
